docs/community/local-groups.mdx

```diff
@@ -93,6 +93,9 @@ us on [Discord](https://discord.com/invite/ktMAKGBnBs) to add your group.
 - [Meshtastic Türkiye Community - Discord](https://discord.gg/ThcG4Arb) 
 - [Meshtastic Türkiye Community - Web (https://trmesh.org)
 
+## Ukraine
+- [WiKi Meshtastic UA](https://wikimesh.pp.ua)
```

Comment:
Thanks for the PR, @olegios! Before this can be merged, we'll need the group to update the logo they are using to conform with our [community group guidelines ](https://meshtastic.org/docs/legal/licensing-and-trademark/#licensing) which require use of the [M PWRD logo](https://meshtastic.org/docs/legal/licensing-and-trademark/#community-project-logos-m-powered--m-pwrd) instead of the official Meshtastic logo

Reply:
Hello!
The logo was updated according to the community group guidelines, please verify.